src/app/components/ListSelector/ListSelector.jsx

```diff
@@ -29,22 +56,29 @@ class ListSelector extends React.Component {
   }
 
   /**
-   * submitFormRequest()
+   * submitFormRequest(submitValue)
    * Submits the request for a new list to the internal server
+   * @param {string} submitValue
    */
-  submitFormRequest() {
+  submitFormRequest(submitValue) {
     // this function will be replaced by submiting to endpoint
-    axios.get(`${config.baseApiUrl}${this.state.submitValue}`)
+    axios.get(`${config.baseApiUrl}${submitValue}`)
       .then(response => {
         // Catch the error from API, and update BookStore back to the default
         if (response.data.statusCode >= 400) {
           this.updateBookStore();
           console.log(
             `API error with status code ${response.data.statusCode}: ${response.data.errorMessage}`
           );
+          // Lead the user to the 404 page
+          this.updateHistory('/books-music-dvds/recommendations/staff-picks/404');
```

Comment:
Instead of updating the history, you should use the router to go to that 404 page.